shared/crossCompilation/rust.md

```diff
@@ -0,0 +1,50 @@
+#### Setting up {{ $language.name }} cross-compilation
```

Comment:
done "__Warning:__ This guide is for Linux and will not work on Windows or OSX!"